man/GGIR.Rd

```diff
@@ -783,6 +781,14 @@ GGIR(mode = \Sexpr{format(formals(GGIR::GGIR)[["mode"]])},
       \item{max_calendar_days}{
       Numeric (default = \Sexpr{GGIR::load_params()[["params_cleaning"]][["max_calendar_days"]]}).
       The maximum number of calendar days to include (set to zero if unknown).}
+      
+      
+      \item{nonWearEdgeCorrection}{
+      Boolean (default = \Sexpr{GGIR::load_params()[["params_cleaning"]][["nonWearEdgeCorrection"]]}).
+        If TRUE then the edges are corrected following description in vanHees2013
```

Comment:
It could be clarified that "edges" refer to the "edges" of the recording, and not to the edges of each non-wear period identified.